### DISCUSSION TITLE
setting dashboard visibility

### DISCUSSION THREAD

**Discussion by publicityreform** [knowledgefutures/pubpub#1589] [Ideas]
this falls under the larger category of making it easier to set and modify pub / collection visibility even after they've been "released" (e.g. toggling between private (contributors only) / community / public)

currently the "dashboard" link appears to offer any visitor to the site (logged in or not) access to all released pubs. 

for us, it would be ideal if this link would not appear for non-community members. 

alternatively, the dashboard view for non-community members would only include "public" releases.

one possible use-case would be making a released pub "hidden" (e.g. adding a draft to a community-only-facing collection for revisions). 

another use-case would allow for moving a pub between private (composing with contributors); community (feedback and revisions); and public (published version).

**Comment by idreyn** [comment]
> currently the "dashboard" link appears to offer any visitor to the site (logged in or not) access to all released pubs, as well as drafts.

We do intentionally expose all released Pubs to visitors through the Dashboard — we find that it's a helpful way to find content — but they should be able to see drafts. If you find that your drafts are visible to users without appropriate membership permissions please let us know!

> one possible use-case would be making a released pub "hidden" (e.g. adding a draft to a community-only-facing collection for revisions).

Totally agree this would be lovely and we have plans on [our roadmap](https://github.com/orgs/pubpub/projects/9#card-59853642) to address it.

**Reply by publicityreform** [reply]
thanks for the quick reply! 

oops - yes - drafts are **not** visible on the dashboard to non-logged-in site visitors (although there is a drafts tab (empty) which is a little confusing. 

if i understand correctly, all drafts can be found and viewed by all community members who are logged in, and all releases can be found / viewed (via the dashboard) by any site visitor?

is there any way to set member permissions for individual pubs / collections? e.g. to give a member "admin" privileges at the scope of a pub or collection: to view, edit, see reviews, and create releases of only the pubs and collections they are working on?
